tests/e2e/volumepropverification_test.go

```diff
@@ -33,14 +33,14 @@ import (
 )
 
 var (
-	esx              string = os.Getenv("ESX")
+	esx               = os.Getenv("ESX")
 	volSizes                = []string{"100MB"}
 	vms                     = []string{os.Getenv("VM1")}
```

Comment:
Not the current change, but why do we create a list here if we just need one VM? Same for volSizes. This has caused 2 unnecessary nested for-loop in TestVolumeProperties().